torch/nn/modules/batchnorm.py

```diff
@@ -435,7 +435,13 @@ class SyncBatchNorm(_BatchNorm):
         >>> m = nn.SyncBatchNorm(100)
         >>> # creating process group (optional)
         >>> # process_ids is a list of int identifying rank ids.
-        >>> process_group = torch.distributed.new_group(process_ids)
+        >>> process_ids = list(range(8))
```

Comment:
nit: maybe call these `ranks` instead of process_ids since it might be confused with OS pids.